networkx/readwrite/tests/test_text.py

```diff
@@ -1,3 +1,4 @@
+import itertools as it
```

Comment:
The abbreviation `it` is too commonly used... That makes it hard to find where the itertools are used (though not that hard). It looks to me like you only use itertools.product.  Maybe you could just use `from itertools import product` or I'm OK with `import itertools` and then use the full name below (those lines aren't very long anyway).